python/doc/examples/probabilistic_modeling/distributions/plot_pointConditionalDistribution.py

```diff
@@ -0,0 +1,255 @@
+"""
+Create a Point Conditional Distribution
+=======================================
+"""
+
+# %%
+# In this example, we create some distributions conditionned by a particular value of its marginals.
+#
+# We consider the following cases:
+# - Case 1:  Bivariate distribution with continuous marginals,
+# - Case 2:  Bivariate distribution with continuous and discrete marginals,
+# - Case 3:  Trivariate distribution with continuous and discrete marginals,
+# - Case 4:  Extreme values conditioning.
+#
+# We illustrate the fact that the range of the conditioned distribution is updated.
+import openturns as ot
+import openturns.viewer as otv
+import openturns.experimental as otexp
+
+ot.ResourceMap.SetAsString("Contour-DefaultColorMapNorm", "rank")
+# sphinx_gallery_thumbnail_number =  5
+
+# %%
+# Case 1: Bivariate distribution with continuous marginals
+# --------------------------------------------------------
+#
+# Here, we consider the bivariate distribution of :math:`(X_0, X_1)` defined by:
+# .. math::
```

Comment:
This is poorly rendered:

![image](https://github.com/user-attachments/assets/2c6bfb96-8f5a-4339-81ad-0bc226a7aa34)